src/app/components/ReadTime/index.tsx

```diff
@@ -97,24 +107,39 @@ export const ReadTimeArticleExperiment = ({
       data-testid="read-time"
     >
       <Text size={fontSize} fontVariant={fontVariant} css={styles.readTimeText}>
-        {readTimeCopyType === 'minutes' ? minutesCopy : quickLongCopy}
+        {copy}
       </Text>
     </div>
   );
 };
 
 export const ReadTime = ({
   readTimeValue,
+  readTimeVariant,
   promoId,
   className,
 }: ReadTimeProps) => {
-  if (isLive() || !readTimeValue) return null;
+  const renderConditions = [
+    !isLive(),
+    readTimeValue,
+    readTimeVariant,
+    readTimeVariant !== 'off',
+  ];
```

Comment:
Super minor, but you could add `.every(Boolean)` at the end of the array which would do the same as the line below.